### DISCUSSION TITLE
typos in documentation

### DISCUSSION THREAD

**Discussion by DanielAjoy** [h3rald/min#164] [Bugs: Fixed]
this page

https://min-lang.org/learn-operators/

says: 

> witb 
> captuee 
> srack

instead of:

> with 
> capture 
> stack

**Comment by DanielAjoy** [comment]
Same document says:

> union stribg|num|quot, and the add method above can be use too sum

it should probably say:

> union str|num|quot, and the add method above can be use **to sum**

and 

> while tbe fullowing 

should probably be:

> while the following

**Reply by celtic-coder** [reply]
Addressed in #165.

**Reply by h3rald** [reply]
Well spotted @DanielAjoy, and thanks @celtic-coder for fixing those.

**Reply by celtic-coder** [reply]
You are welcome! 😄

**Comment by DanielAjoy** [comment]
Trying not to pollute the discussions with more typos, I'll mention them here:

This page https://min-lang.org/learn-operators/ mentions this example which contains an extra backquote at the end

``
!"ls -al"`
``

a little further down it contains the visible text

``
[help`](/reference-lang#op-help) symbol.
``

**Reply by h3rald** [reply]
Fixed, thanks! 😊👍

**Comment by DanielAjoy** [comment]
> Creates a new quotation quot2 obtaining by selecting all elements of quot1 between indexes int1 and int2.

Should say:

> Creates a new quotation quot2 **obtained** by selecting all elements of quot1 between indexes int1 and int2.
